packages/bcer-data-portal/app/src/hooks/useLocation.tsx

```diff
@@ -0,0 +1,90 @@
+import { BusinessLocation } from '@/constants/localInterfaces';
+import { routes } from '@/constants/routes';
+import { AppGlobalContext } from '@/contexts/AppGlobal';
+import { formatError } from '@/util/formatting';
+import React, { useContext, useEffect, useState } from 'react';
+import { useHistory } from 'react-router';
+import { useAxiosGet } from './axios';
+
+function useLocation(locationIds?: string) {
+  const history = useHistory();
+  const [selectedLocations, setSelectedLocations] = useState<
+    BusinessLocation[]
+  >([]);
```

Comment:
this looks strange being on multiple lines

Reply:
agreed! not sure why prettier is breaking line this way